src/systems/systemItemsDialog.component.tsx

```diff
@@ -48,15 +117,63 @@ const SystemItemsDialog = React.memo((props: SystemItemsDialogProps) => {
   const { mutateAsync: moveItemsToSystem, isPending: isMovePending } =
     useMoveItemsToSystem();
 
+  const errorUsageStatuesItemId = usageStatuesErrors
```

Comment:
Yeah, it would be good if we make a note somewhere, its best if we can avoid filters on the front end where possible as they will get slower the more data there is and I think some are unnecessary.

Reply:
#480